Index: app/src/main/java/com/example/bridge/testCommit.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/example/bridge/testCommit.java	(date 1581798040000)
+++ app/src/main/java/com/example/bridge/testCommit.java	(date 1581798040000)
@@ -0,0 +1,4 @@
+package com.example.bridge;
+
+public class testCommit {
+}
